### PR TITLE
Remove all local imports from PZ

### PR DESCRIPTION
**Pull request by pseudo-rnd-thoughts**

### CODE REVIEW

File: pettingzoo/classic/go/go.py
Comment (elliottower) [inline pettingzoo/classic/go/go.py:1]: Is pygame not necessary here? I see calls using it so I think it should probably be added back

File: test/all_parameter_combs_test.py
Comment (elliottower) [inline test/all_parameter_combs_test.py:1]: This is just because we only test on some of the modules, correct?

**Review by elliottower — COMMENTED** [review]
Looks mostly good besides a few questions. The __all__ parts are a good catch, should’ve definitely been there before

File: pettingzoo/classic/go/go.py
Comment (pseudo-rnd-thoughts) [reply pettingzoo/classic/go/go.py:1]: No, as it is already imported in the module

File: test/all_parameter_combs_test.py
Comment (pseudo-rnd-thoughts) [reply test/all_parameter_combs_test.py:1]: Im not sure what your question is? I just replaced the `*` with the actual environments

Comment (elliottower) [reply test/all_parameter_combs_test.py:1]: Makes sense, I was pretty sure that was it but just wanted to make sure you weren’t doing something else

**Review by elliottower — APPROVED** [review]

**Pull request merged**